src/Shared/Telemetry/TelemetryProcessor.ts

```diff
@@ -8,123 +8,124 @@ import { userContext } from "../../UserContext";
 /**
  * Class that persists telemetry data to the portal tables.
  */
-// TODO: Move to a separate Diagnostics folder
-// TODO: Log telemetry for StorageExplorer case/other clients as well
-export default class TelemetryProcessor {
-  public static trace(action: Action, actionModifier: string = ActionModifiers.Mark, data?: any): void {
-    sendMessage({
-      type: MessageTypes.TelemetryInfo,
-      data: {
-        action: Action[action],
-        actionModifier: actionModifier,
-        data: JSON.stringify(data)
-      }
-    });
-
-    appInsights.trackEvent({ name: Action[action] }, TelemetryProcessor.getData(data));
-  }
 
-  public static traceStart(action: Action, data?: any): number {
-    const timestamp: number = Date.now();
-    sendMessage({
-      type: MessageTypes.TelemetryInfo,
-      data: {
-        action: Action[action],
-        actionModifier: ActionModifiers.Start,
-        timestamp: timestamp,
-        data: JSON.stringify(data)
-      }
-    });
-
-    appInsights.startTrackEvent(Action[action]);
-    return timestamp;
-  }
+export function trace(action: Action, actionModifier: string = ActionModifiers.Mark, data?: unknown): void {
```

Comment:
Can we use the fat arrow syntax?
```suggestion
export const trace = (action: Action, actionModifier: string = ActionModifiers.Mark, data?: unknown): void => {
```